examples/cfd/01_convection.ipynb

```diff
@@ -263,7 +261,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "1.0*(-dt*h_x*u(t, x, y) + dt*h_x*u(t, x, y - h_y) - dt*h_y*u(t, x, y) + dt*h_y*u(t, x - h_x, y) + h_x*h_y*u(t, x, y))/(h_x*h_y)\n"
+      "dt*(-1.0*u(t, x, y)/h_y + 1.0*u(t, x, y - h_y)/h_y - 1.0*u(t, x, y)/h_x + 1.0*u(t, x - h_x, y)/h_x + u(t, x, y)/dt)\n"
```

Comment:
suspiciously many more divisions...we must ensure they're captured in the generated code (this is again the issue about "lack of dedicated factorization tests" hitting us?)

Reply:
TTI opcount test still all good. I think this is marginal but I;ll double check.